code/controllers/subsystems/voting/poll_types.dm

```diff
@@ -17,18 +28,29 @@
 	multiple_votes = TRUE //Duel votes are fun
 	can_revote = TRUE
 	can_unvote = TRUE //In case you heck up
+	only_admin = TRUE
 
 	see_votes = FALSE //No swaying
+	minimum_win_percentage = 0.6
+	var/non_admin = TRUE
 
-/*To prevent abuse and rule-by-salt, the evac vote weights each player's vote based on a few parameters
-	If you are alive and have been for a while, then you have the normal 1 vote
-	If you are dead, or just spawned, you get only 0.6 votes
-	If you are an antag or a head of staff, you get 1.2 votes
-*/
-#define VOTE_WEIGHT_LOW	0.6
-#define VOTE_WEIGHT_NORMAL	1
-#define VOTE_WEIGHT_HIGH	1.2 //To tie 2 dead votes but not over-rule 2 living
-#define MINIMUM_VOTE_LIFETIME	15 MINUTES
+/datum/poll/restart/proc/lower_minium()
+	minimum_win_percentage -= 0.1
+
+	if(minimum_win_percentage <= 0.2)
+		SSticker.shift_end(15 MINUTES)
+		for (var/mob/M as mob in SSmobs.mob_list)
+			to_chat(M, "<br><center><span class='danger'><b><font size=4>The round is ending do to exceeding 8 hours.</font></b><br></span></center><br>")
```

Comment:
```suggestion
		for(var/mob/M as mob in SSmobs.mob_list)
			to_chat(M, "<br><center><span class='danger'><b><font size=4>The round is ending due to exceeding 8 hours.</font></b><br></span></center><br>")
```
Why are there linebreaks? to_chat should, by its nature, put the message on its own line.